src/libnvme-mi.map

```diff
@@ -1,6 +1,11 @@
 LIBNVME_MI_1_12 {
 	global:
 		nvme_mi_mi_xfer;
+		nvme_mi_get_aem_fd;
+		nvme_mi_enable_aem;
+		nvme_mi_aem_process;
+		nvme_mi_disable_aem;
+		nvme_mi_aem_get_next_event;
 };
```

Comment:
Having all of these together highlights a bit of naming inconsistency: some are verb_scope, others are scope_verb. We generally have scope_verb (but where the verb may sometimes be a nvme command name, so it can get a bit mixed up!).

Given a scope of `nvme_mi_aem`, I would suggest:

```
	nvme_mi_aem_get_fd;
	nvme_mi_aem_enable;
	nvme_mi_aem_process;
	nvme_mi_aem_disable;
	nvme_mi_aem_get_next_event;
```